_inc/client/components/notice/style.scss

```diff
@@ -77,14 +77,21 @@
 	}
 }
 
-.dops-notice__content {
+.dops-notice__content.dops-notice__content {
```

Comment:
It seems like a CSS class duplication.

Reply:
No this is intentional to increase the specificity of the selector on the page.

Reply:
Other wise a different selector can removed the underline of the link. Which makes the link not visible at all. 
```
.jetpack-pagestyles a {
	text-decoration: none;
}```

Reply:
TIL :)